pkg/graph/resolvers/model_plan.go

```diff
@@ -20,6 +21,11 @@ func ModelPlanCreate(logger *zap.Logger, modelName string, store *storage.Store,
 		},
 	}
 
+	err := BaseStructPreCreate(logger, plan, principal, store, false)
```

Comment:
I know this it nit-picky, but can you leave a comment here explaining why access control is set to false? That way it's clear to anyone reading that it's disabled here because its the initial model plan, so of _course_ (obvious to us) you can't check for an existing collaborator without an error.

Reply:
Thanks! Sounds good @ClayBenson94, change made and pushed.